polkadot/node/core/pvf/src/security.rs

```diff
@@ -14,155 +14,247 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::LOG_TARGET;
-use std::path::Path;
+use crate::{Config, SecurityStatus, LOG_TARGET};
+use futures::join;
+use std::{fmt, path::Path};
 use tokio::{
 	fs::{File, OpenOptions},
 	io::{AsyncReadExt, AsyncSeekExt, SeekFrom},
 };
 
-/// Check if we can sandbox the root and emit a warning if not.
+const SECURE_MODE_ANNOUNCEMENT: &'static str =
+	"In the next release this will be a hard error by default.
+     \nMore information: https://wiki.polkadot.network/docs/maintain-guides-secure-validator#secure-validator-mode";
```

Comment:
I would also add a note about the new flag that will enable validators to run "unsecurely". They should know that this is will be an option

Reply:
The wiki link does mention it. I would rather validators read the full information, instead of being given a flag here and applying it without thinking. When we have the error it will spell out the exact flag here, but maybe it shouldn't...

Reply:
The error should spell out the entire flag. However, it is fine to wait until this is a hard error.

Reply:
Alright cool. Thanks for the review!